docs/src/components/action/InfoCard.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import Link from 'docs/src/modules/components/Link';
+
+interface InfoCardProps {
+  icon: React.ReactNode;
+  title: string;
+  description: string;
+  link?: string;
+}
+
+export default function InfoCard({ icon, title, description, link }: InfoCardProps) {
+  return (
+    <Paper
+      component={link ? Link : 'div'}
+      href={link}
+      noLinkStyle={Boolean(link)}
```

Comment:
I pushed a quick fix on HEAD. This line creates this warning on http://0.0.0.0:3000/.

<img width="708" alt="Screenshot 2023-10-19 at 16 22 20" src="https://github.com/mui/material-ui/assets/3165635/289ba843-7911-41f1-81bc-9a722df10e68">

The fix: 031131c20c3793f8d3319454d4dce8e4b647ffbd.

Reply:
Thank you!